Kconfig

```diff
@@ -38,14 +38,6 @@ config HOST_PTABLE
 	bool
 	default n
 
-config TASK_HAVE_PRIORITY_MEDIUM
```

Comment:
Regarding the commit message part about "priority and deadline". The deadline determines the priority in the system with io tasks scheduled using EDF discipline. And there is no other one. However we may talk about multiple level scheduling we have in SOF (instead of priority combinations/low latency terms etc.):
- first level: round robin till completion,
- second level (within one of the first level's task): EDF pre-emptively.

Otherwise, I am not sure how we may keep high/medium/low levels together with EDF and prove the system will be stable if the total load requirement is met.

Reply:
Rephrased commit description. Priority is not used in the terms of usual low/medium/high scheduling, but to differentiate tasks which have some meaningful deadline and the ones that don't e.g. main firmware loop is scheduled as idle task, so it can be chosen as the last task to execute.